lib/services/HDInsight2/lib/hDInsightManagementClient.js

```diff
@@ -4094,7 +4094,17 @@ var ClusterOperations = ( /** @lends ClusterOperations */ function() {
                     var kindInstance = kindValue;
                     clusterDefinitionInstance.clusterType = kindInstance;
                   }
-                  
+
+                  var componentVerionsInDefinition = clusterDefinitionValue['componentVersion'];
+                  if (componentVerionsInDefinition !== null && componentVerionsInDefinition !== undefined) {
+                      var componentVerionsInstance = {};
+                      for (var item in componentVerionsInDefinition) {
+                        componentVerionsInstance[item] = componentVerionsInDefinition[item];
+                      }
+
+                      clusterDefinitionInstance.componentVerions = componentVerionsInstance;
+                  }
+
```

Comment:
@voidfunction - This library is autogenerated. Moreover this is autogenerated from the old hydra-specs using the old codegenerator hydra.
The spec that you are referring to is the new swagger spec. This library needs to be autogenerated using the new code generator Autorest.